### PR TITLE
raidboss: add 1st ex trial targetable lines to timeline

### PR DESCRIPTION
**Pull request by Legends0**
Not sure if astral eclipse timings vary or not, but I was seeing ~12.7 and ~13.7 in fflogs.

### CODE REVIEW

File: ui/raidboss/data/06-ew/trial/zodiark-ex.txt [hunk 1]
```diff
@@ -59,7 +60,9 @@ hideall "Infernal Torrent"
 # Star tethers
 314.4 "--sync--" sync / 1[56]:[^:]*:Zodiark:67E9:/ window 300,10
 325.9 "Apomnemoneumata" sync / 1[56]:[^:]*:Zodiark:6D7B:/
+336.9 "--targetable--"
 347.6 "Astral Eclipse" sync / 1[56]:[^:]*:Zodiark:67C3:/
+360.3 "--targetable--"
```
Comment (quisquous) [inline ui/raidboss/data/06-ew/trial/zodiark-ex.txt:65]: Are these by hand, or from make_timeline with `-it Zodiark`?

From my log, I get:
```
325.9 "Apomnemoneumata" sync / 1[56]:[^:]*:Zodiark:6D7B:/
336.5 "--targetable--"
347.6 "Astral Eclipse" sync / 1[56]:[^:]*:Zodiark:67C3:/
347.6 "--untargetable--"
359.7 "--targetable--"
```

and

```
535.7 "Astral Eclipse" sync / 1[56]:[^:]*:Zodiark:67C3:/
535.7 "--untargetable--"
547.8 "--targetable--"
```

Comment (Legends0) [reply ui/raidboss/data/06-ew/trial/zodiark-ex.txt:65]: these are by hand, so you may have more accurate logs

Comment (Legends0) [reply ui/raidboss/data/06-ew/trial/zodiark-ex.txt:65]: In network log I see 12.0 seconds for first eclipse to when it is targetable.

Comment (Legends0) [reply ui/raidboss/data/06-ew/trial/zodiark-ex.txt]: The adds spawning I looked at the network log for when they appear after Opheos Eidolon.

**Comment by quisquous** [comment]
Thanks for the followup!

**Pull request merged**